src/content/docs/plugin/single-instance.mdx

```diff
@@ -96,10 +96,15 @@ By default, when you initiate a new instance while the application is already ru
 use tauri::{AppHandle, Manager};
 
 pub fn run() {
-    tauri::Builder::default()
-        .plugin(tauri_plugin_single_instance::init(|app, args, cwd| {
+    let mut builder = ;tauri::Builder::default()
```

Comment:
```suggestion
    let mut builder = tauri::Builder::default()
```

Reply:
just checking if you're actually reviewing or just clicking 🤣